include/nfl/prng/FastGaussianNoise.hpp

```diff
@@ -23,6 +20,10 @@
 #include <boost/math/tools/roots.hpp>
 #endif
 
+namespace nfl {
+
+#define tstbit(x, n)  ((x << (63 - n)) >> 63 )
+
```

Comment:
This does not make sense : namespaces don't affect macros. What we should do here is make tstbit an inlined constexpr function, as in:

template<class T0, class T1>
constexpr inline auto tstbit(T0 x, T1 n) -> decltype((x << (63 - n)) >> 63 ) { return ((x << (63 - n)) >> 63 ); }
```